master/getting-started/docker/installation/vagrant-coreos/index.md

```diff
@@ -59,9 +59,29 @@ And finally check that Docker is running on both hosts by running
 
     docker ps
 
-## 2. Try out Calico networking
+## 2. Install Calico
 
-Now you have a basic two node CoreOS cluster setup, see the [Calico with Docker Networking Walkthrough]({{site.baseurl}}/{{page.version}}/getting-started/docker/tutorials/basic) for information on how to try Calico Networking.
+With your VM's running, and connectivity between them established,
+it is time to launch `calico/node`.
+
+The Vagrant install will automatically place `calicoctl` in
```

Comment:
This is too much detail.  

"The Vagrant machines already have calicoctl installed"